recipe/meta.yaml

```diff
@@ -40,12 +40,12 @@ test:
     - ansi2html
   commands:
     - pip check
-    - nosetests tests/test_ansi2html.py
+    - nosetests tests/test_ansi2html.py  # [py<312]
```

Comment:
This is interesting.  It looks like upstream may not use nose (tox really) and if they don't we could probably just use regular pytest.